bigbluebutton-html5/imports/api/2.0/users/server/handlers/changeRole.js

```diff
@@ -1,34 +1,9 @@
-import Logger from '/imports/startup/server/logger';
 import { check } from 'meteor/check';
-import Users from '/imports/api/2.0/users';
+import changeRole from '../modifiers/changeRole';
 
 export default function handleChangeRole({ body }, meetingId) {
-  const { userId, role, changedBy } = body;
+  check(body, Object);
+  check(meetingId, String);
 
-  check(userId, String);
-  check(role, String);
-  check(changedBy, String);
-
-  const selector = {
-    meetingId,
-    userId,
-  };
-
-  const modifier = {
-    $set: {
-      role,
-    },
-  };
-
-  const cb = (err, numChanged) => {
-    if (err) {
-      return Logger.error(`Changed user role: ${err}`);
-    }
-
-    if (numChanged) {
-      return Logger.info(`Changed user role ${role} id=${userId} meeting=${meetingId} by changedBy=${changedBy}`);
-    }
-  };
-
-  return Users.update(selector, modifier, cb);
+  changeRole({ body }, meetingId);
```

Comment:
I can see you extract body from the payload, `... default function handleChangeRole({ body },...`
Then check the object and wrap it into the object again.
Why don't you omit this step?

E.g.
`export default function handleChangeRole(payload, meetingId) {`
...
`check(payload.body, Object);`
...
`changeRole(payload, meetingId);`